src/main/java/aeon/controller/Command.java

```diff
@@ -0,0 +1,297 @@
+
+    public static final String CREATE_FILE_FAILED = "Failed to create file to store task!";
+
+    private static ArrayList<String> rawDescList = new ArrayList<>();
+    public static final String TASK_DELETED = "Task deleted!";
+
+    public static void CommandProcessor() {
+        printWelcomeMessage();
+        //Task[] list = new Task[100];
+        ArrayList<Task> list = new ArrayList<>();
+        readSavedTaskList(list);
+        Scanner in = new Scanner(System.in);
+        String response = in.nextLine();
+        while (!response.equals("bye")) {
+            String[] words = response.split(" ", 2);
+            switch (words[0]) {
+            case ("list"):
+                printListOfTasks(list);
+                break;
+            case ("unmark"):
+                try {
+                    unmarkTask(list, words);
+                } catch (IndexOutOfBoundsException e) {
+                    System.out.println(TASK_NOT_FOUND);
+                } catch (NumberFormatException e) {
+                    System.out.println(INVALID_INTEGER_MSG);
+                }
+                break;
+            case ("mark"):
+                try {
+                    markTask(list, words);
+                } catch (IndexOutOfBoundsException e) {
+                    System.out.println(TASK_NOT_FOUND);
+                } catch (NumberFormatException e) {
+                    System.out.println(INVALID_INTEGER_MSG);
+                }
+                break;
+            case ("todo"):
+                try {
+                    addTodoTask(list, words);
+                } catch (IndexOutOfBoundsException e) {
+                    System.out.println(TODO_DESC_ERROR);
+                }
+                break;
+            case ("deadline"):
+                try {
+                    addDeadlineTask(list, words);
+                } catch (IndexOutOfBoundsException e) {
+                    System.out.println(DEADLINE_FORMAT_ERR);
+                }
+                break;
+            case ("event"):
+                try {
+                    addEventTask(list, words);
+                } catch (IndexOutOfBoundsException e) {
+                    System.out.println(EVENT_FORMAT_ERR);
+                }
+                break;
+            case("delete"):
+                try {
+                    deleteTask(list, words);
+                } catch (IndexOutOfBoundsException e) {
+                    System.out.println(TASK_NOT_FOUND);
+                } catch (NumberFormatException e) {
+                    System.out.println(INVALID_INTEGER_MSG);
+                }
+                break;
+            default:
+                System.out.println(INVALID_COMMAND);
+                break;
+            }
+            response = in.nextLine();
+        }
+        printGoodbyeMessage();
+    }
+
+    private static void readSavedTaskList(ArrayList<Task> list) {
+        File f = new File(FILE_PATH);
+        File d = new File(DIR_PATH);
```

Comment:
Can consider a more appropriate name to improve readability (e.g. `FILE`, `DIRECTORY`).